running/ReplacingDocker/podman.sh

```diff
@@ -0,0 +1,311 @@
+    ps -ef | grep -v grep | grep --color=auto 200000
+    echo ""
+
+    read -p "--> clear"
+    clear
+}
+
+forkexec() {
+    # Podman Fork/Exec model
+    read -p "Podman Fork/Exec Model"
+    echo ""
+
+    read -p "--> cat /proc/self/loginuid"
+    cat /proc/self/loginuid
+    echo ""
+    echo ""
+
+    read -p "--> sudo podman run -ti fedora bash -c \"cat /proc/self/loginuid; echo\""
+    sudo podman run -ti fedora bash -c "cat /proc/self/loginuid; echo"
+    echo ""
+
+    read -p "--> sudo docker run -ti fedora bash -c \"cat /proc/self/loginuid; echo\""
+    sudo docker run -ti fedora bash -c "cat /proc/self/loginuid; echo"
+    echo ""
+
+    read -p "--> sudo auditctl -w /etc/shadow"
+    sudo auditctl -w /etc/shadow 2>/dev/null
+    echo ""
+
+    # Showing how podman keeps track of the person trying to wreak havoc on your system
+    read -p "--> sudo podman run --privileged -v /:/host fedora touch /host/etc/shadow"
+    sudo podman run --privileged -v /:/host fedora touch /host/etc/shadow
+    echo ""
+
+    read -p "--> ausearch -m path -ts recent -i | grep touch | tail -n 1 | grep --color=auto 'auid=[^ ]*'"
+    sudo ausearch -m path -ts recent -i | grep touch | tail -n 1 | grep --color=auto 'auid=[^ ]*'
+    echo ""
+
+    read -p "--> sudo docker run --privileged -v /:/host fedora touch /host/etc/shadow"
+    sudo docker run --privileged -v /:/host fedora touch /host/etc/shadow
+    echo ""
+
+    read -p "--> ausearch -m path -ts recent -i | grep touch | tail -n 1 | grep --color=auto 'auid=[^ ]*'"
+    sudo ausearch -m path -ts recent -i | grep touch | tail -n 1 |grep --color=auto 'auid=[^ ]*'
+    echo ""
+
+    read -p "--> clear"
+    clear
+}
+
+top() {
+    # Podman top commands
+    read -p "Podman top features"
+    echo ""
+
+    read -p "--> sudo podman run -d fedora sleep 1000"
+    sudo podman run -d fedora sleep 1000
+    echo ""
+
+    read -p "--> sudo podman top --latest pid hpid"
+    sudo podman top --latest pid hpid
+    echo ""
+
+    read -p "--> sudo podman top --latest label"
+    sudo podman top --latest label
+    echo ""
+
+    read -p "--> sudo podman top --latest seccomp"
+    sudo podman top --latest seccomp
+    echo ""
+
+    read -p "--> sudo podman top --latest capeff"
+    sudo podman top --latest capeff
+    echo ""
+
+    read -p "--> clear"
+    clear
+}
+
+pods() {
+    sudo podman kill -a
+    sudo podman rm -a -f
+    clear
+
+    read -p "--> sudo podman pod list"
+    sudo podman pod list
+    echo ""
+
+    read -p "--> sudo podman pod --name podtest create"
+    sudo podman pod create --name podtest
+    echo ""
+
+    read -p "--> sudo podman pod create --pod podtest -d fedora sleep 600"
+    sudo podman create --pod podtest -d fedora sleep 600
+    echo ""
+
+    read -p "--> sudo podman pod create --pod podtest -d fedora sleep 600"
+    sudo podman create --pod podtest -d fedora sleep 600
+    echo ""
+
+    read -p "--> sudo podman ps"
```

Comment:
This just shows a blank line, do you want to add a `-a` to the ps command here?

Reply:
Added text, goal is to show that there are no containers running, but after you start the pod the two containers are running. when you stop the pod, the containers are not running.